packages/network_info_plus/network_info_plus/example/integration_test/network_info_plus_test.dart

```diff
@@ -34,6 +67,20 @@ void main() {
       expect(_networkInfo.getWifiSubmask(), isNotNull);
       expect(_networkInfo.getWifiGatewayIP(), isNotNull);
       expect(_networkInfo.getWifiBroadcast(), isNotNull);
-    });
+    }, skip: !Platform.isAndroid);
```

Comment:
This is probably not necessary now, because this test should work on _all_ platforms.